src/inabox/inabox-viewport.js

```diff
@@ -262,35 +262,23 @@ export class ViewportBindingInabox {
   }
 
   /** @override */
-  getLayoutRectAsync(el) {
+  getGlobalClientRect() {
     if (!this.requestPositionPromise_) {
       this.requestPositionPromise_ = new Promise(resolve => {
         this.iframeClient_.requestOnce(
             MessageType.SEND_POSITIONS, MessageType.POSITION,
```

Comment:
Is it clear from this message what kind of position this is? Position to me sounds like document position, which is not exactly the case here now?

Reply:
So, to confirm, is this a "client rect" position? Because from the name it doesn't sound like that.

Reply:
I agree it's not super clear. What we have now is a position object that contains
```
{
  targetRect: // This return clientRect of the root
  viewportRect: // This returns {scrollLeft, scrollTop, width, height} of the global viewport
}
```
I think we might be able to remove `viewportRect` here in near future we can confirm the value is not used and we separate `Viewport` and `Root` class.